### ISSUE TITLE
Code in Markdown displays superflous ```

### ISSUE THREAD

**Issue by duerrsimon** [gradio-app/gradio#4452]
### Describe the bug

![image](https://github.com/gradio-app/gradio/assets/16979577/5a4b64e1-6c2a-48a5-ae94-42fd37016365)

For a code in `gr.Markdown` if specified with three backticks the backticks are rendered at the end in the UI 

### Is there an existing issue for this?

- [X] I have searched the existing issues

### Reproduction

```
gr.Markdown("""```test
```""")
```

### Logs

```shell
NA
```

### System Info

```shell
latest gradio version 3.33.1
```

### Severity

annoying

**Comment by freddyaboulton** [comment]
I think this is fixed on 3.34.0 - can you please verify @duerrsimon ?

```python
import gradio as gr

with gr.Blocks() as demo:
    gr.Markdown("""```test
    this is code
    ```""")

demo.launch()
```

<img width="1011" alt="image" src="https://github.com/gradio-app/gradio/assets/41651716/f067e419-5e4e-4143-8492-a0fa1770694d">

**Comment by duerrsimon** [comment]
Yes fixed for me too!
